docs/build/references/command-line-interface.md

```diff
@@ -309,28 +309,59 @@ The default C-Chain config is:
 
 ```json
 {
-  "snowman-api-enabled": false,
-  "coreth-admin-api-enabled": false,
-  "coreth-performance-api-enabled": false,
-  "net-api-enabled": true,
-  "rpc-gas-cap": 50000000,
-  "rpc-tx-fee-cap": 100,
-  "eth-api-enabled": true,
-  "personal-api-enabled": false,
-  "tx-pool-api-enabled": false,
-  "debug-api-enabled": false,
-  "web3-api-enabled": true,
-  "local-txs-enabled": false,
-  "pruning-enabled": false,
-  "api-max-duration": 0, // Default to no maximum
-  "api-max-blocks-per-request": 0, // Default to no maximum
-  "allow-unfinalized-queries": false,
-  "log-level": "info"
+   "snowman-api-enabled":false,
```

Comment:
nit: let's keep it to an indent-size of two spaces and add a space between the key and value